### PR TITLE
chore(github): Update issue template to use external main che repository

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Provide guidance on how to create new issues (user should use main Eclipse Che repository)

### What issues does this PR fix or reference?
Part of Endgame https://github.com/eclipse/che/issues/13637

### CODE REVIEW

**Comment by benoitf** [comment]
If KO, let me know the expected wording

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [inline .github/ISSUE_TEMPLATE/where-to-report-issues.md]:
```suggestion
All issues for Eclipse Che are managed at [https://github.com/eclipse/che/issues](https://github.com/eclipse/che/issues).
```

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [inline .github/ISSUE_TEMPLATE/where-to-report-issues.md]:
```suggestion
name: "⚠️ Where to report issues?"
```

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [inline .github/ISSUE_TEMPLATE/where-to-report-issues.md]:
```suggestion
about: File issues in the main Eclipse Che repository at [https://github.com/eclipse/che/issues](https://github.com/eclipse/che/issues).
```

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [inline .github/ISSUE_TEMPLATE/where-to-report-issues.md]:
```suggestion
title: Issues need to be filed in the main Eclipse Che repository
```

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [inline .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]:
```suggestion
🚨 Please don't submit new issues that are not related to documentation in this repository. 🚨
```

**Review by rkratky — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (slemeur) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: @rkratky : getting consistent with the others Che repositories, we want to have a single place to consolidate the backlog and make it easier to the community to look for issues/feedbacks/enhancements. There is a kind/docs label on eclipse/che repo which should be used appropriately.

Comment (rkratky) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: @slemeur I find it highly unintuitive to file issues about the content of a repo in a different repo. It also prevents us from fully using GH features that simplify work on PRs and issues when handled 'normally', i.e. through the issue queue of respective repos.

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: here I would still say that it's "no issues at all" as it was decided that all issues should be in `eclipse/che` repository

Issues should have `area/doc` label in `eclipse/che`

**Comment by benoitf** [comment]
thanks @rkratky for your review, I've accepted all your changes except one being discussed

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: > it was decided

Where?

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: @rkratky at the F2F during End Game discussion
All issues have to be handled in only one repository: Eclipse Che (https://github.com/eclipse/che)

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md]: Here I can't merge the link syntax as the about section is not written in markdown so user will see ugly stuff
I've fixed my error related to `on`/`in`

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md]: you can see how it's rendered on https://github.com/che-incubator/chectl/issues/new/choose

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (rkratky) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: > at the F2F during End Game discussion

Heh, I must've dozed off... :(

>  All issues have to be handled in only one repository: Eclipse Che (https://github.com/eclipse/che)

OK, I get it, thanks (I find that to be an unfortunate decision, but too late now).

Will somebody be send an e-mail to che-dev about this?

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: I'm not sure it was acted to send a separate mail but yes I can send it

do you think it's better after merging all these `github redirect` PR (and on other repositories) or I should send it as soon as possible ?

Comment (rkratky) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: I'd send it as soon as possible. It's a significant process change.

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: mail sent

**Comment by benoitf** [comment]
@rkratky do you still require some changes ?

**Review by rkratky — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/where-to-report-issues.md
Comment (slemeur) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: The whole point is simplicity and consistency for the community and users. 
If someone wants to create an issue, there will be a single backlog which give an easier way to get visibility on what's going on. The use of labels and milestone will be easier to manage too. 
If someone is looking at all the issues resolved for a given milestone, until having a single backlog, he/she would have to do a request across dozen of repositories. 
It is also very inconvenient when someone is looking at if an issue already exist.

One last point, we currently have issues for docs on github.com/eclipse/che repo, I'm doubtful we've been able to manage these two spread backlogs in sync.

Comment (rkratky) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: I understand the reasoning (and I agree with it). But I believe the wrong tool was chosen. GH issues are specifically tailored for working with respective repositories. Tracking issues for multiple repos in one issue queue doesn't scale. As a general-purpose issue tracker, GH issues sucks.

For such purpose, an external issue tracker should be used. GH issues is severely limited in functionality -- for example, the inability to filter issue notification mails in any way makes it impossible track project areas efficiently. One always needs to open the list of issues on the web just to check if new issues were created for a particular label.

Comment (slemeur) [reply .github/ISSUE_TEMPLATE/where-to-report-issues.md:14]: That's a fair point @rkratky. 
For the committers it will require a bit more effort, but for the community and users, it will become cleaner and simpler.

**Pull request merged**
